src/_guides/language/sound-faq.md

```diff
@@ -104,14 +104,20 @@ an expression evaluates to a specific type at runtime.
 Strong mode performs stronger type checking. For more information, see
 [Strong mode vs. checked mode](/guides/language/sound-dart#strong-mode-vs-checked-mode).
 
-Checked mode will be obsolete in Dart 2.0 and will be removed. For more
-information, see [Dart 2.0 Updates.](/dart-2.0)
+{% include checked-mode-2.0.html %}
+
+{% comment %}
+update-for-dart-2
+{% endcomment %}
 
 <a name="why-another-mode"></a>
 ### Why do we need another mode for Dart?
 
-Checked mode will be obsolete in Dart 2.0 and will be removed. For more
-information, see [Dart 2.0 Updates.](/dart-2.0)
+{% include checked-mode-2.0.html %}
```

Comment:
It's weird to have only a note and no content for a FAQ answer. I'd delete this question.